packages/genesys-spark-components/src/components/beta/gux-time-zone-picker/gux-time-zone-picker.tsx

```diff
@@ -76,11 +76,14 @@ export class GuxTimeZonePickerBeta {
   }
 
   private filterTimeZoneList(timeZoneList: GuxTimeZoneOption[]) {
-    const searchString = this.searchString;
     return timeZoneList.filter(tzOption => {
-      return this.getFormattedTimeZoneOption(tzOption)
-        .toLowerCase()
-        .includes(searchString.toLowerCase());
+      return tzOption.mainCities.some(city => {
+        return tzOption.displayTextName
+          .concat(tzOption.baseDisplayOffsetText)
+          .concat(city)
+          .toLowerCase()
+          .includes(this.searchString.toLowerCase());
```

Comment:
Does this filter only work for english?

Reply:
Ye I tested it through Dutch and French and the filter still works as intended 👍

Reply:
As discussed on zoom the city names need to be localized for this to work as expected and the may be a huge localisation effort .